papers/doc/7-systems/sm8521.md

```diff
@@ -0,0 +1,16 @@
+# Sharp SM8512
+
+The SM8512 is the CPU and sound chip of the Game.com, a handheld console released in 1997 as a competitor to the infamous Nintendo Virtual Boy.
+
+Ultimately, most of the games for the Game.com ended up being failiures in the eyes of reviewers, thus giving the Game.com a pretty bad reputation. This was one of the reasons that the Game.com only ended up selling at least 300,000 units. For these reasons and more, the Game.com ended up being discontinued in 2000. 
```

Comment:
And any others in where it says 12

Reply:
There were no other typos where it had that issue.